.gitignore

```diff
@@ -2,6 +2,8 @@
 lib/*
 /tetragon
 /tetragon-alignchecker
+/tetragon-tester
+/tetragon-vmtests-run
```

Comment:
Why is this needed? Typically, these utilities are build iin tests/vmtests.

See: https://github.com/cilium/tetragon/blob/main/tests/vmtests/.gitignore

Reply:
removed, let me know if I should rebase / force-push